### PR TITLE
Numbered Pagination Added

### PR DESCRIPTION
**Pull request by JarrydLong**
Relates to #95

When on Page 1 
pagination-pt-1

After moving further along the page numbers

pagination-pt-2

### CODE REVIEW

File: templates/directory.php [hunk 1]
```diff
@@ -604,6 +604,33 @@ function pmpromd_shortcode($atts, $content=null, $code="")
...
+
+		$number_of_pages = $totalrows / $limit;
+		//Page Numbers
+		?>
+			<span class='pmpro_page_numbers'>
+		<?php
+		$counter = 0;
+		if( empty( $pn ) || $pn != 1 ){
+			echo '<a href="'.esc_url(add_query_arg( $query_args, get_permalink($post->ID))).'" title="'.__( 'Previous', 'pmpromd' ).'">...</a>';
+		}
```
Comment (sc0ttkclark) [inline templates/directory.php:616]:
```suggestion
		if ( empty( $pn ) || $pn != 1 ) {
			echo '<a href="' . esc_url( add_query_arg( $query_args, get_permalink( $post->ID ) ) ) . '" title="' . esc_attr__( 'Previous', 'pmpromd' ) . '">...</a>';
		}

```

File: templates/directory.php [hunk 2]
```diff
@@ -604,6 +604,33 @@ function pmpromd_shortcode($atts, $content=null, $code="")
...
+			if( $counter <= 6 ){
+				$query_args = array(
+					'ps' => $s,
+					'pn' => $i,
+					'limit' => $limit,
+				);
+
+				if( $i == $pn ){ $active_class = 'class="pmpro_page_active"'; } else { $active_class = ''; }
+				
+				echo '<a href="'. esc_url(add_query_arg( $query_args, get_permalink($post->ID))).'" '.$active_class.' title="'.sprintf( __('Page %s', 'pmpromd' ), $i ).'">'.$i.'</a>';
```
Comment (sc0ttkclark) [inline templates/directory.php:627]:
```suggestion
				echo '<a href="' . esc_url( add_query_arg( $query_args, get_permalink( $post->ID ) ) ) . '" ' . $active_class . ' title="' . esc_attr( sprintf( __('Page %s', 'pmpromd' ), $i ) ) . '">' . $i . '</a>';
```

**Review by sc0ttkclark — CHANGES_REQUESTED** [review]
Minor tweak but there's some escaping to do

**Comment by andrewlimaza** [comment]
Merging in as items have been escaped.

**Pull request merged**